src/server/utils/nsfs_schema_utils.js

```diff
@@ -0,0 +1,34 @@
+/* Copyright (C) 2023 NooBaa */
+'use strict';
+
+const RpcError = require('../../rpc/rpc_error');
+const { default: Ajv } = require('ajv');
+const ajv = new Ajv({ verbose: true, allErrors: true });
+const { KEYWORDS } = require('../../util/schema_keywords');
+const common_api = require('../../api/common_api');
+
+ajv.addKeyword(KEYWORDS.methods);
+ajv.addKeyword(KEYWORDS.doc);
```

Comment:
I think that a better location for this file can be under manage_nsfs folder

Reply:
Do you mean where 'manage.nsfs.js' is now? to move this file to `src/cmd`?

Reply:
no, please put this file under manage_nsfs folder